docs/tutorials/update-user-email.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+---
+
+# Tutorial: Update a user's email
+
+In this tutorial, you'll learn the operations to call to update a user's email.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+Make sure you've completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you'll use for the tutorial.
+
+## Update a user's email
+
+You can update an existing user's email in the To-Do service. To do so, `PUT` an [`email`](../api/user/id/email) resource containing the user's details.
```

Comment:
```suggestion
You can update an existing user's email in the To-Do service.
```
The 2nd sentence might be confusing as the procedure that follows looks like you need to do more than that.